### ISSUE TITLE
ValueError: Unknown game version: 4.1.4. Known versions: ['latest'].

### ISSUE THREAD

**Issue by GJHall** [yalidu/liir#3]
I have been able to get SMAC installed and working to train COMA/QMIX agents.

When I tried to run your code the first time I received this error:

>File "src/main.py", line 33, in my_main
>     env_args['seed'] = _config["seed"]
> sacred.utils.SacredError: The configuration is read-only in a captured function!

To fix this I added the following code to the main.py file:
`SETTINGS['CONFIG']['READ_ONLY_CONFIG'] = False`

now I am receiving:
> ValueError: Unknown game version: 4.1.4. Known versions:  ['latest'].

This is the same error I receive when I try to run the Variance Base Control code from [](https://github.com/saizhang0218/VBC)

Any ideas?

Thank you for your time.

Edit: I am using version 4.11.3

**Comment by yalidu** [comment]
Hi, @GJHall , I have not met errors on the ` _config['seed']`. This repo is developed based on pysc2=2.0.2. Pls make sure that the version of SMAC is also compatible.  

As to the game version, I met the same value error before, but this disappears after I set the game version in `sc2.yaml` consistent with the version of the SC2 game core that is installed on the server.

Hope these will help! Thanks!

**Comment by GJHall** [comment]
I think the issue has to do with:

A) Windows
B) s2client-proto not offering version 4.11.x hashed ([](https://github.com/Blizzard/s2client-proto/blob/master/buildinfo/versions.json))

I tried changing the sc2.yaml file to the right version but I do not think windows looks at this config line. Still get the same error.

I will try playing around with it on my linux system.

Closing the issue and will update if I find a solution.

Thank you for your quick response.
